packages/webapp/src/components/collection/cardCollectionDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import Image from 'next/image'
+import { Card } from 'src/store/types' 
+import { MintDeckModal } from 'src/components/modals/mintDeckModal'
+
+interface CardCollectionDisplayProps {
+  cards: Card[];
+  isHydrated: boolean;
+  setSelectedCard: (card: Card | null) => void;
+  onCardToggle: (card: Card) => void;
+  selectedCards: Card[]; 
+}
+
+const CardCollectionDisplay: React.FC<CardCollectionDisplayProps> = ({ cards, isHydrated, refetch, setSelectedCard, selectedCards, onCardToggle, isEditing }) => {
+  return (
+    <>
+      <style jsx>{`
+        .card-in-deck {
+          box-shadow: 0 0 10px orange;
+        }
+      `}</style>
+
+      <div className="col-span-7 flex rounded-xl border overflow-y-auto">
+        {isHydrated && cards.length === 0 && (
+          <div className="flex flex-row w-full justify-center items-center">
+            <MintDeckModal callback={refetch} />
+          </div>
+        )}
+
+        {isHydrated && cards.length > 0 && (
+          <div className="grid grid-cols-4 overflow-y-auto pb-4">

```

Comment:
I know this was there before, but the `grid-cols-4` + the rest means that as we resize the window all cards resize, up to very small sizes

It might be good to change that to simply display as much as we need.

Okay, after much trial & error, querying chat GPT and the tailwind / CSS docs:

- Replace the grid line with 
`<div className="flex flex-wrap justify-around overflow-y-auto pb-4">`
- Add `grow w-[220px] max-w-[330px]` to the card div.
- Add `className="aspect-square"` to the image
- Remove `style={{ height: 'fit-content' }}`
    
This causes elements to grow dynamically from 220 to 300px, and if they reached max size, the remaining space is put around the elements in a balanced way (`justify-around`). The `aspect-square` thing simply replaces the `fit-content` line in a way that is clearer.

The image stuff ensures the image resizes